<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWindow</class>
 <widget class="QMainWindow" name="MyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>663</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sync-my-L²P</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/Icons/Icons/magnifier.png</normaloff>:/Icons/Icons/magnifier.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="browserTab">
       <attribute name="title">
        <string>Dateibrowser</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QGroupBox" name="dataBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Daten</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="refreshPushButton">
             <property name="text">
              <string>aktualisieren</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QGroupBox" name="synchronisationBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Synchronisation</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="addSelectionPushButton">
             <property name="text">
              <string>Auswahl hinzufügen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeSelectionPushButton">
             <property name="text">
              <string>Auswahl entfernen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="openDownloadfolderPushButton">
             <property name="text">
              <string>Downloadordner öffnen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="syncPushButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>synchronisieren</string>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="viewBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Darstellung</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="showNewDataPushButton">
             <property name="text">
              <string>neue Dateien anzeigen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="expandPushButton">
             <property name="text">
              <string>alle expandieren</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="contractPushButton">
             <property name="text">
              <string>alle kontrahieren</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="sizeLimitCheckBox">
             <property name="text">
              <string>Dateigrößenfilter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sizeLimitSpinBox">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="suffix">
              <string> MB</string>
             </property>
             <property name="prefix">
              <string>maximal </string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="dateFilterCheckBox">
             <property name="text">
              <string>Datumsfilter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="minDateEdit">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="displayFormat">
              <string>'ab' dd.MM.yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="maxDateEdit">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="displayFormat">
              <string>'bis' dd.MM.yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QGroupBox" name="searchBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Suche</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="searchLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>Suchbegriff</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="searchPushButton">
             <property name="text">
              <string>Suchen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" rowspan="5">
         <widget class="QTreeView" name="dataTreeView">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="optionsTab">
       <attribute name="title">
        <string>Einstellungen</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="userDataBox">
          <property name="title">
           <string>Benutzerdaten</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QCheckBox" name="userDataSaveCheckBox">
             <property name="text">
              <string>Logindaten speichern</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="userNameLineEdit">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>Benutzername</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="userPasswordLineEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>Passwort</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="loginPushButton">
               <property name="text">
                <string>Login</string>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="downloadFolderBox">
          <property name="title">
           <string>Downloadverzeichnis</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLineEdit" name="downloadFolderlineEdit">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>Downloadverzeichnis</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="downloadFolderPushButton">
             <property name="text">
              <string>Durchsuchen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="filterOptionsBox">
          <property name="title">
           <string>Dokumentenfilter</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="2">
            <widget class="QCheckBox" name="documentsCheckBox">
             <property name="text">
              <string>Dokumente</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="sharedMaterialsCheckBox">
             <property name="text">
              <string>Strukturierte Materialien</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="currentSemesterCheckBox">
             <property name="text">
              <string>aktuelles Semester einbinden</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="oldSemesterCheckBox">
             <property name="text">
              <string>frühere Semester einbinden</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QCheckBox" name="tutorDocumentsCheckBox">
             <property name="text">
              <string>Dokumente für Betreuer</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="2">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QCheckBox" name="exercisesCheckBox">
             <property name="text">
              <string>Übungen</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="automatisationBox">
          <property name="title">
           <string>Automatisieren</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QCheckBox" name="autoLoginOnStartCheckBox">
             <property name="text">
              <string>Automatisch Einloggen beim Programmstart</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="autoSyncOnStartCheckBox">
             <property name="text">
              <string>Automatisch Synchronisieren beim Programmstart</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="minimizeInTrayCheckBox">
             <property name="text">
              <string>Beim Minimieren im Tray verschwinden</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="autoBackgroundSyncCheckBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Automatisch Synchronieren</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="autoBackgroundSyncSpinBox">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="suffix">
                <string> Minuten</string>
               </property>
               <property name="prefix">
                <string>alle </string>
               </property>
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>999</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="updateBox">
          <property name="title">
           <string>Update</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="0">
            <widget class="QPushButton" name="updatePushButton">
             <property name="text">
              <string>nach Updates suchen</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="versionNumberLabel">
             <property name="text">
              <string>Version: 1.0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Kontakt</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Offizielle Webseite: &lt;html&gt; &lt;a style = 'text-decoration:none'href ='link'&gt;http://www.Sync-my-L2P.de&lt;/a&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Offizielle Facebookpage: &lt;html&gt; &lt;a style = 'text-decoration:none'href ='link'&gt;http://www.Facebook.de/Sync-my-L2P&lt;/a&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Quellcode: &lt;html&gt; &lt;a style = 'text-decoration:none'href ='link'&gt;https://github.com/Sync-my-L2P/Sync-my-L2P&lt;/a&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logTab">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListView" name="logListView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
